platforms/hyperledger-fabric/charts/invoke_chaincode/templates/invoke_chaincode.yaml

```diff
@@ -128,19 +160,36 @@ spec:
         command: ["sh", "-c"]
         args:
         - |-
-          #!/bin/bash
+          #!/bin/bash sh
+          set -x
           export ARGS='{"Args":'"'["${INSTANTIATION_ARGUMENTS}"]'"'}'
           export qARGS="'${ARGS}'"
-          touch ./invokeChaincode.sh
-          version1_4=`echo $NETWORK_VERSION | grep -c 1.4`
-          if [ $version1_4 = 1 ];then
```

Comment:
The same chart should be supporting both fabric version 1.4.x and 2.x, so this patch should be there